### PR TITLE
[shared] enable OS-specific command handling in helm charts

### LINKED ISSUE #2366: feat(shared): ensure OS specific commands in helm charts are handled properly
### Description
As a developer I want to ensure that helm charts have less of OS specific commands and proper conditions to support different OS.  

### Acceptance Criteria
1. TDB

### PR DESCRIPTION
**Pull request by saurabhkumarkardam**
### **Commit to be reviewed**
---
**feat(shared): enable OS-specific command handling in helm charts**

```
This PR improves the handling of OS-specific commands within Helm charts.

Changes:
- Introduced a script to check the Operating System and its supporting package manager for efficient package installation.
- Added a ConfigMap object to insert the same above-mentioned script into the container, improving flexibility and compatibility.

Additional change:
- Updated the StorageClass Helm chart to resolve the node affinity issue.

There are only 3 platforms that are currently using OS-Specific Command in Helm Charts directly, and this PR is also made for these 3 platforms only:
- Quorum
- Hyperledger-Fabric
- Substrate
```

fixes #2366

### CODE REVIEW

**Comment by jagpreetsinghsasan** [comment]
Where is the `bevel-vault.sh` script located in our repo?

**Comment by saurabhkumarkardam** [comment]
> @saurabhkumarkardam where is the `bevel-vault.sh` script located in our repo?

platforms/shared/charts/bevel-scripts/scripts/bevel-vault.sh

File: platforms/hyperledger-fabric/charts/catools/templates/deployment.yaml [hunk 1]
```diff
@@ -133,6 +133,9 @@ spec:
       - name: scripts-volume
         configMap:
           name: bevel-vault-script
+      - name: packet-manager
```
Comment (sownak) [inline platforms/hyperledger-fabric/charts/catools/templates/deployment.yaml:136]: Please rename to 'package-manager'

File: platforms/hyperledger-fabric/charts/generate_cacerts/templates/job.yaml [hunk 2]
```diff
@@ -126,8 +129,12 @@ spec:
           command: ["sh", "-c"]
           args:
             - |-
-              apk update && apk add jq curl openssl;
-              source /scripts/bevel-vault.sh
+              . /scripts/bevel-vault.sh
+              . /scripts/packet-manager.sh
```
Comment (sownak) [inline platforms/hyperledger-fabric/charts/generate_cacerts/templates/job.yaml:133]: rename

**Review by sownak — CHANGES_REQUESTED** [review]
just rename packet to package

File: platforms/hyperledger-fabric/charts/catools/templates/deployment.yaml (hunk 1 above)
Comment (saurabhkumarkardam) [reply platforms/hyperledger-fabric/charts/catools/templates/deployment.yaml]: Fixed. Please review.

File: platforms/hyperledger-fabric/charts/generate_cacerts/templates/job.yaml (hunk 2 above)
Comment (saurabhkumarkardam) [reply platforms/hyperledger-fabric/charts/generate_cacerts/templates/job.yaml]: Fixed. Please review.

**Review by sownak — APPROVED** [review]

**Review by suvajit-sarkar — APPROVED** [review]
LGTM

**Pull request merged**